frontend/catalyst/device/qjit_device.py

```diff
@@ -309,7 +309,7 @@ def __init__(self, original_device):
         super().__init__(wires=original_device.wires, shots=original_device.shots)
```

Comment:
Should we not remove the shots from the parent constructor (since I'm assuming PL is intending to remove that kwarg eventually?

Reply:
Right now it will be in a co-existing phase.

Reply:
Any reason not to remove it already? Note that I'm not suggesting we remove it from the QJITDevice class, just from the qml.devices.Device class.

Reply:
https://github.com/PennyLaneAI/catalyst/pull/1952/commits/a0da493a973496a93941b489fa0d5531898bdddb